src/types.ts

```diff
@@ -132,3 +134,12 @@ export interface LeaderboardHeader {
   key: string;
   label: string;
 }
+
+export type AllOrderings = GitHubOrderBy | LeetCodeOrderBy | ApplicationOrderBy;
+export type AllLeaderboardData =
```

Comment:
nit: I prefer interface here, can just be `LeaderbordEntry` which other types implement, and then you can change AllLeaderboardData to LeaderboardEntry[]

Reply:
Fixed